openjdk/excludes/ProblemList_openjdk11-openj9.txt

```diff
@@ -201,6 +213,12 @@ java/nio/channels/DatagramChannel/Promiscuous.java https://github.com/AdoptOpenJ
 java/nio/channels/SocketChannel/AdaptSocket.java https://github.com/eclipse/openj9/issues/4317 macosx-all
 java/nio/file/Files/CopyAndMove.java https://github.com/AdoptOpenJDK/openjdk-tests/issues/795 macosx-all
 jdk/nio/zipfs/ZipFSTester.java https://github.com/eclipse/openj9/issues/4679 linux-x64,macosx-all
+java/nio/channels/Selector/OutOfBand.java https://github.com/AdoptOpenJDK/openjdk-tests/issues/2422 z/OS-s390x
```

Comment:
According to your comment in https://github.com/AdoptOpenJDK/openjdk-tests/issues/2422, You are excluding `java/nio/channels/Selector/OutOfBand.java` temporarily due to openj9-openjdk-jdk11-zos/issues/575. So, The PR for the exclusion should have openj9-openjdk-jdk11-zos/issues/575 as the cause of this temporary exclusion, not https://github.com/AdoptOpenJDK/openjdk-tests/issues/2422. I believe https://github.com/AdoptOpenJDK/openjdk-tests/issues/2422 is just an issue you created for triage, it should not be used as a reason for an exclusion.

Reply:
Hi @Mesbah-Alam - I think in a previous PR from @psoujany I had requested that we ideally reference open issues for exclusions (so that when we automate the checking of them, the automation script can access them).  Though it is fine to reference the internal issue in the placeholder issue.  (I don't like that we have duplicate issue, but I do not see a way around it, and I will hope that the temporary placeholder issues are not long-lived).

Reply:
@ShelleyLambert Hi Shelley, oh.. I wasn't aware of that automation :) .. In that case I am fine with using the open issue in the exclude file! @pdbain-ibm , please Ignore my comment about using openj9-openjdk-jdk11-zos/issues/575!

Reply:
@Mesbah-Alam - Its _upcoming_ automation (related: https://github.com/AdoptOpenJDK/openjdk-tests/issues/1853 and https://github.com/AdoptOpenJDK/openjdk-tests/issues/2368), but thought to enforce the use of full and openly accessible URLs to so that when we do finish the automation, we do not have to update as many playlists and problemlists.

Reply:
@ShelleyLambert - makes sense Shelley!